homeassistant/components/roborock/__init__.py

```diff
@@ -35,7 +36,13 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         user_data, {device.duid: RoborockDeviceInfo(device) for device in devices}
     )
     network_info = {
-        device.duid: await mqtt_client.get_networking(device.duid) for device in devices
+        device.duid: result
+        for device, result in zip(
```

Comment:
Now that its bigger and more than one line the style prefers not using a comprehension.

Reply:
That change better?